i3_resurrect/treeutils.py

```diff
@@ -88,8 +88,14 @@ def get_workspace_tree(workspace):
             if container['type'] != 'con':
                 pass
             for ws in container['nodes']:
-                if ws['name'] == workspace:
-                    return ws
+                # Select workspace and trigger name and num field
+                if workspace.isdigit():
+                    if 'num' in ws:
+                        if ws['num'] == int(workspace):
+                            return ws
+                else:
+                    if ws['name'] == workspace:
```

Comment:
Similarly here I would opt for `elif` instead of an extra unnecessary indentation level

Reply:
Ok, it's clearer indeed. 
I updated pull request.

Also, Your plugin is great, thanks very much for your work!